aea/cli/generate.py

```diff
@@ -45,44 +50,62 @@ def generate(
 
 @generate.command()
 @click.argument("protocol_specification_path", type=str, required=True)
+@click.option(
+    "--l",
+    "language",
+    type=str,
+    required=False,
+    default="python",
+    help="Specify the language in which to generate the protocol package.",
+)
 @pass_ctx
-def protocol(ctx: Context, protocol_specification_path: str) -> None:
+def protocol(ctx: Context, protocol_specification_path: str, language: str) -> None:
     """Generate a protocol based on a specification and add it to the configuration file and agent."""
-    _generate_item(ctx, PROTOCOL, protocol_specification_path)
```

Comment:
Reintroduce that separation! It helps reuse of the function outside the cli.

Reply:
Done